src/components/Header/Header.jsx

```diff
@@ -4,6 +4,7 @@ import Typography from "@mui/material/Typography";
 import { Stack } from "@mui/material";
```

Comment:
Hey can we stick to using single quotes for strings please?

It looks like this component had double quotes before, but most of the components in the app have single quotes, so we'll try and keep it consistent :)